torch/package/_selective_extern.py

```diff
@@ -0,0 +1,14 @@
+SELECTIVE_INTERN_LIST = [
+    $interned_modules
+]
+
+if "__torch_package__" in dir():
+
+    def __getattr__(name):
+        package_name = $package_name
+        if name not in SELECTIVE_INTERN_LIST:
+            return getattr(_extern_copy, name)
+        obj = locals().get(name, None)
+        obj = obj if obj else globals().get(name, None)
+        __import__(f'{package_name}.{name}',level=0)
```

Comment:
you can do something like `_torch_package_importer.import_module()` for a higher level API.

Reply:
Uhh can clarify what you mean here? Do you mean make a new function called _torch_package_importer?

Reply:
Oh I meant that the PackageImporter installs itself onto a global called `_torch_package_importer` (see: https://github.com/pytorch/pytorch/blob/master/torch/package/package_importer.py#L100)

And we can use that to import the module.

Reply:
oh yes I remember that, yeah will add that in instead. It'll make things more clear. Is there another advantage to using that over just __import__?

Reply:
in this case it doesn't really make a difference, but `__import__` is more low-level and is not supposed to be called directly from user code (that's what `import_module` is for)